client/components/select-dropdown/style.scss

```diff
@@ -31,17 +30,22 @@ $compact-header-height: 35;
 	.accessible-focus &:focus,
 	.accessible-focus .select-dropdown.is-open & {
 		z-index: z-index( 'root', '.accessible-focus .select-dropdown.is-open .select-dropdown__container' );
-		box-shadow: 0 0 0 2px $blue-light;
 		.select-dropdown__header {
 			border-color: $blue-wordpress;
 		}
 	}
 
+	.accessible-focus & {
```

Comment:
Yes, I think it's ok.
It should be something like (plain CSS):

```css
.accessible-focus .select-dropdown__container {
    border-radius: 4px;
}
```

Without this rule, when you open the dropdown using the keyboard, the external outline does not match the perimeter of the component, and looks weird.